Airports/EGBK/Active_Runway.txt

```diff
@@ -0,0 +1,2 @@
+ACTIVE_RUNWAY:EGBK:23R:1
+ACTIVE_RUNWAY:EGBK:23R:0
```

Comment:
Still does not match the runways available

Reply:
I don't get it, it matches the format of other airports, e.g. LL - `ACTIVE_RUNWAY:EGLL:27L:1
ACTIVE_RUNWAY:EGLL:27R:0`. Do you want me to include 03L?

Reply:
Needs to be a runway in the Runway.txt file - there you define 21R/03L only -> so you cannot have an active runway of 23R

Reply:
Oh, clearly I can't read sorry about that one! Will solve that now